samples-generator/bin/3d-tiles-samples-generator.js

```diff
@@ -1027,7 +1071,7 @@ function createTileset() {
 
     var tilesetJson = {
         asset : {
-            version : '0.0',
+            version : versionNumber,
```

Comment:
`createTilesetJsonSingle` is still using version `0.0`.

All the other tilesets look good though.